lib/kramdown/converter/math_engine/sskatex/details.rb

```diff
@@ -0,0 +1,264 @@
+# -*- coding: utf-8 -*-
+#
+#--
+# Copyright (C) 2017 Christian Cornelssen <ccorn@1tein.de>
+#
+# This file is part of kramdown which is licensed under the MIT.
+#++
+
+# Defer automatic JS engine selection
+ENV_EXECJS_RUNTIME = ENV['EXECJS_RUNTIME']
+begin
+  ENV['EXECJS_RUNTIME'] = 'Disabled'
+  require 'execjs'
+ensure
+  ENV['EXECJS_RUNTIME'] = ENV_EXECJS_RUNTIME
+end
+
+require 'json'
+
+# Consider this a lightweight alternative to MathjaxNode.
```

Comment:
I have derived the webgen-based documentation for SsKaTeX from the Rdoc, not the other way around, because
* `ri Kramdown::Converter::MathEngine::SsKaTeX` works in a text-based terminal, does not need a web browser.
* Keeping the documentation in sync with the implementation is easier when both are in the same source file.

That said, it's clearly a duplication and thus makes maintenance more expensive, and I agree that by now the user-visible documentation should be the definitive one. I'll deduplicate or move some parts of the module's rdoc to the internal functions or objects that implement them.